### PR TITLE
chore: bump hubble sdk

### PR DESCRIPTION
**Pull request by Andrei997**
**Description**
We released a new feature in Hubble SDK that allows users to login from Jupyter Notebook and Google Colab using Personal Access Tokens, or using the browser, through a convenient UI. Because of this, we released `jina-hubble-sdk>=0.20.2`. 
This PR bumps the required of hubble sdk to that version. 

**This change should not directly affect any of the functionality of Jina core.** 

For the UI, see the images attached below.

_Login UI_
Screen Shot 2022-10-14 at 22 05 43

_Login via browser UI (after clicking the `Browser login` button)_
Screen Shot 2022-10-13 at 14 15 39

_Success UI_
Screen Shot 2022-10-13 at 14 21 55

_Error UI_
Screen Shot 2022-10-13 at 14 21 21

**Goals:**
- bump hubble sdk version to latest version

**Tasks**
- [x] check and update documentation

### CODE REVIEW

File: extra-requirements.txt [hunk 1]
```diff
@@ -33,7 +33,7 @@ grpcio-health-checking>=1.46.0,<1.48.1:  core
 pyyaml>=5.3.1:              core
 packaging>=20.0:            core
 docarray>=0.16.4:           core
-jina-hubble-sdk>=0.19.0:    core
+jina-hubble-sdk>=0.20.2:    core
```
Comment (mapleeit) [inline extra-requirements.txt:36]:
```suggestion
jina-hubble-sdk==0.20.2:    core
```

File: jina/resources/extra-requirements.txt [hunk 2]
```diff
@@ -33,7 +33,7 @@ grpcio-health-checking>=1.46.0,<1.48.1:  core
 pyyaml>=5.3.1:              core
 packaging>=20.0:            core
 docarray>=0.16.4:           core
-jina-hubble-sdk>=0.19.0:    core
+jina-hubble-sdk>=0.20.2:    core
```
Comment (mapleeit) [inline jina/resources/extra-requirements.txt:36]:
```suggestion
jina-hubble-sdk==0.20.2:    core
```

**Review by mapleeit — DISMISSED** [review]
Shall we pin the version of `jina-hubble-sdk` so that the new feature wouldn't automatically be introduced to the user? And we can control when to anounce it to the user and add it to the release notes of Core. 

Otherwise it's good to go IMO @JoanFM . The description is well written, good job @Andrei997

**Comment by JoanFM** [comment]
> Shall we pin the version of `jina-hubble-sdk` so that the new feature wouldn't automatically be introduced to the user? And we can control when to anounce it to the user and add it to the release notes of Core.
> 
> Otherwise it's good to go IMO @JoanFM . The description is well written, good job @Andrei997

I agree with this approach @mapleeit , let's pin it @Andrei997

**Comment by Andrei997** [comment]
Alrighty, sounds like a good idea.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**